.orchestra/ci/install-system-dependencies.sh

```diff
@@ -107,6 +107,16 @@ PACKAGES+=(ripgrep)
 #
 PACKAGES+=(perl)
 
+#
+# Needed by `revng mass-testing test-harness`
+#
+PACKAGES+=(time)
+
+#
+# Needed by `revng mass-testing generate-report` to convert .svg to .png
+#
+PACKAGES+=(inkscape)
```

Comment:
Please let's make this an optional depedency. This would be the first dependency that pulls in GTK. NOGO.

Reply:
Ok, can I at least `which('inkscape')` in `revng mass-testing generate-report`?

Reply:
Works.

Reply:
Done